std/haxe/macro/Compiler.hx

```diff
@@ -246,7 +249,7 @@ class Compiler {
 	}
 
 	/**
-		Load a type patch file that can modify declared classes fields types
+		Load a type patch file that can modify the field types within declared classes and enums.
```

Comment:
Basing this on [extract.patch](https://github.com/HaxeFoundation/haxe/blob/development/extra/extract.patch), which has enums.